ui/src/components/popovers/PasteModal.tsx

```diff
@@ -9,12 +9,12 @@ import { TextAreaWithLabel } from "@components/TextArea";
 import { SettingsPageHeader } from "@components/SettingsPageheader";
 import { useJsonRpc } from "@/hooks/useJsonRpc";
 import { useHidStore, useRTCStore, useUiStore, useSettingsStore } from "@/hooks/stores";
-import { keys, modifiers } from "@/keyboardMappings";
-import { layouts, chars } from "@/keyboardLayouts";
+import { KeyStroke, keys, modifiers } from "@/keyboardMappings";
+import { KeyboardLayout, selectedKeyboard } from "@/keyboardLayouts";
 import notifications from "@/notifications";
 
-const hidKeyboardPayload = (keys: number[], modifier: number) => {
-  return { keys, modifier };
+const hidKeyboardPayload = (modifier: number, keys: number[]) => {
```

Comment:
Swapped the order around because that's what it really gets send as in the HID message.
See page 60 of the [HID spec](https://www.usb.org/sites/default/files/documents/hid1_11.pdf)